pkg/codec/vaapi/vp8.go

```diff
@@ -315,7 +317,7 @@ func (e *encoderVP8) Read() ([]byte, func(), error) {
 
 	e.frParam.data.framerate = C.uint(e.rate.Calc())
 
-	if kf {
+	if kf || e.forceKeyFrame {
```

Comment:
`forceKeyFrame` isn't reset, so every frames after `ForceKeyFrame()` will be keyframe.